userland/tools/elf2tbf/src/main.rs

```diff
@@ -330,18 +260,13 @@ fn do_work(
     // so that changes to the app won't move it.
     let appstate_offset = app_start_offset as u32;
     let appstate_size = appstate.shdr.size as u32;
-    let relocation_data_offset = align8!(appstate_offset + appstate_size);
     // Make sure we pad back to a multiple of 8.
```

Comment:
Need to either fix below back to `align8` (correct I think), or update this comment to 4